workflow.md

```diff
@@ -316,8 +318,26 @@ Following steps should be executed to release version `X = Maj.Min.Fix`:
 
 6. Issues from version to be set status *Waiting for test* and assigned to QA team lead.
 
-7. QA team to perform consise testing of version (release note testing or regression testing, depending on case)
+7. If minor version wasn't yet published to end users, version to be deployed on QAnet cluster
+   and QA team to perform testing there
+
+#### Release deploy & publishing
+
+If minor version is to be initially published to end user or is published already, but fix was prepared and to be delivered to end users, following steps should be executed.
+
+1. Prepare installers to test
+  a. If version contains changes needed to be deployed on testnet cluster
+      1) Testnet cluster should be cloned (with databases, same keys) and version should be deployed on clone
+      2) Installers should be prepared to work with this clone
+  b. If version contains no changes which require redeploy, installers just to be rebuild to use testnet cluster
+
+2. QA team to perform consise testing of version (release note testing or regression testing, depending on case)
+
+3. QA team to approve no blockers discovered (i.e. only minor issues/glitches remain)
 
-8. QA team to approve no blockers discovered (i.e. only minor issues/glitches remain)
+4. In case redeploy of testnet is needed
+    a. Redeploy should be performed
+    b. Installers for testnet rebuild
+    c. QA team should confirm installers work well on testnet
```

Comment:
Please include requirements for updating Explorer as well. My understanding is it must use the same version as deployed on testnet.

DevOps needs to know operational requirements, like the minimum time window the two services are allowed to have a version mismatch.